src/PlusDataCollection/Andor/vtkPlusAndorVideoSource.cxx

```diff
@@ -57,9 +57,9 @@ PlusStatus vtkPlusAndorVideoSource::ReadConfiguration(vtkXMLDataElement* rootCon
 
   Shutter = static_cast<ShutterMode>(std::atoi(deviceConfig->GetAttribute("Shutter")));
   m_AcquisitionMode = static_cast<vtkPlusAndorVideoSource::AcquisitionMode>(std::atoi(deviceConfig->GetAttribute("AcquisitionMode")));
+  m_ReadMode = static_cast<vtkPlusAndorVideoSource::ReadMode>(std::atoi(deviceConfig->GetAttribute("ReadMode")));
```

Comment:
If "ReadMode" is absent from configuration file, this will default to 0 meaning FullVerticalBinning. This is undesirable.

Reply:
would something like this be better?
```
if(deviceConfig->GetAttribute("ReadMode")
{
  m_ReadMode = static_cast<vtkPlusAndorVideoSource::ReadMode>(std::atoi(deviceConfig->GetAttribute("ReadMode")));
}
else
{
  m_ReadMode = ReadMode::Image;
}
```

and same with `m_AcquisitionMode` and `m_TriggerMode`?

Reply:
Yes. It is more verbose, but more convenient to use. Maybe even like:
```
const char * readModeString = deviceConfig->GetAttribute("ReadMode");
if(readModeString)
{
  m_ReadMode = static_cast<vtkPlusAndorVideoSource::ReadMode>(std::atoi(readModeString));
}
else
{
  m_ReadMode = ReadMode::Image;
}
```